init.sql

```diff
@@ -0,0 +1,3 @@
+GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
```

Comment:
Granting ALL PRIVILEGES to a user can pose a security risk if not properly managed. Consider limiting privileges to only those necessary for the application's functionality to enhance security.